core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsFeedName.java

```diff
@@ -141,13 +158,16 @@ public boolean equals(Object other) {
       return true;
     }
     if (other instanceof GtfsFeedName) {
-      return this.countryFirstName.equals(((GtfsFeedName) other).countryFirstName);
+      return Objects.equals(this.countryFirstName, ((GtfsFeedName) other).countryFirstName);
     }
     return false;
   }
 
   @Override
   public int hashCode() {
+    if (countryFirstName == null) {
+      return 0;
```

Comment:
> the hash code of a non-null argument and 0 for a null argument


According to https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hashCode-java.lang.Object-

Reply:
That is just the behavior of that static method because given `null` it can't compute any other relevant hashcode for the given value. In this case, you would want to return `this.hashCode()` for the hashcode of the object instance so you can tell if two instances are the same.